src/io.jl

```diff
@@ -81,6 +74,21 @@ function create_parameters_and_sets_from_file(input_folder::AbstractString)
     return params, sets
 end
 
+"""
+    get_df(path, file_name, schema)
```

Comment:
This needs to be updated to match the signature below

Reply:
Got it!